sequelizeDatabase/sequelFunctions.js

```diff
@@ -1,350 +1,481 @@
-    return models.EmployeeHistory.create({
-        //TODO: Add a restriction so you can't have Individuals and companies with Mismatched FundID's.
-        UserID: UserID,
-        IndividualID: IndividualID,
-        CompanyID: CompanyID,
-        PositionName: PositionName,
-        StartDate: StartDate,
-        EndDate: EndDate
-    }).then((history) =>{
-        //console.log('History Added: ', history);
-        return history;
-    }).catch(err => console.error('Error in insertEmployeeHistory',err));
-}
+  return models.EmployeeHistory.create({
+    //TODO: Add a restriction so you can't have Individuals and companies with Mismatched FundID's.
+    UserID: UserID,
+    IndividualID: IndividualID,
+    CompanyID: CompanyID,
+    PositionName: PositionName,
+    StartDate: StartDate,
+    EndDate: EndDate
+  })
+    .then(history => {
+      //console.log('History Added: ', history);
+      return history;
+    })
+    .catch(err => console.error("Error in insertEmployeeHistory", err));
+};
 
-insertFromCsvLine = (UserID, FundID, PortfolioCompanyName, EmployeeName, OriginalPostion, OriginalStartDate, OriginalEndDate, CurrentEmployer, CurrentPostion, LinkedInUrl, Comments) =>{
-    console.log("\n\n Fields are:, ", UserID, FundID, PortfolioCompanyName, EmployeeName, OriginalPostion, OriginalStartDate, OriginalEndDate, CurrentEmployer, CurrentPostion, LinkedInUrl, Comments, "\n");
-    insertPerson(FundID, EmployeeName, LinkedInUrl, Comments).then((newPerson) => {
-        insertCompany(PortfolioCompanyName, FundID).then((originalCompany) => {
-            insertOriginalFundPosition(newPerson.IndividualID, originalCompany.CompanyID, OriginalPostion).then(() => {
-                //NOTE: I'm hoping it doesn't freak out that I included no arguements here.
-                insertEmployeeHistory(UserID, newPerson.IndividualID, originalCompany.CompanyID, OriginalPostion, OriginalStartDate, OriginalEndDate).then(() => {
-                     insertCompany(CurrentEmployer,FundID).then((newCompany) => {
-                        insertEmployeeHistory(UserID, newPerson.IndividualID, newCompany.CompanyID, CurrentPostion, OriginalStartDate, OriginalEndDate)
-                     });
-                });
-            }).catch(err => console.error(">>ERROR ini insertFromCSVLine, in insertOriginal Position."));
-        });
-    }).catch(err => console.error('Error in "insertFromCsvLine", ', err));
-    
-}
+insertFromCsvLine = (
+  UserID,
+  FundID,
+  PortfolioCompanyName,
+  EmployeeName,
+  OriginalPostion,
+  OriginalStartDate,
+  OriginalEndDate,
+  CurrentEmployer,
+  CurrentPostion,
+  LinkedInUrl,
+  Comments
+) => {
+  console.log(
+    "\n\n Fields are:, ",
+    UserID,
+    FundID,
+    PortfolioCompanyName,
+    EmployeeName,
+    OriginalPostion,
+    OriginalStartDate,
+    OriginalEndDate,
+    CurrentEmployer,
+    CurrentPostion,
+    LinkedInUrl,
+    Comments,
+    "\n"
+  );
+  insertPerson(FundID, EmployeeName, LinkedInUrl, Comments)
```

Comment:
This function had its layout changed (not edited) by an extension called "Prettier". (Actually, all functions were reformated by this extension.)